packs/bestiary-family-ability-glossary/ghost-draining-touch.json

```diff
@@ -11,7 +11,7 @@
         },
         "category": "offensive",
         "description": {
-            "value": "<p>With a touch, the ghost attempts to drain a living creature's life force. It makes a ghostly hand Strike but deals no damage on a hit.</p>\n<p>Instead, the target becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} for 1 day, and the ghost regains [[/r (max(1,floor(@actor.level/2)))[healing]]]{HP equal to half its own level}.</p>"
+            "value": "<p>With a touch, the ghost attempts to drain a living creature's life force. It makes a ghostly hand Strike but deals no damage on a hit.</p>\n<p>Instead, the target becomes @UUID[Compendium.pf2e.conditionitems.Item.Drained]{Drained 1} for 1 day, and the ghost regains @Damage[(max(1,floor(@actor.level/2)))[healing]].</p>"
```

Comment:
Should we restore the label `{HP equal to half its own level}`? Without it we divert from the source material and it's unclear what the rules are?